components/02-molecules/callout/_yds-callout.twig

```diff
@@ -14,10 +14,12 @@
       heading__blockname: callout__base_class,
       heading: callout__heading,
     } %}
-    {% include "@atoms/typography/text/yds-text.twig" with {
-      text__blockname: callout__base_class,
-      text__content: callout__text,
-    } %}
+    {% if callout__text %}
```

Comment:
This works in Storybook, but doesn't actually fix it in Drupal.

Since the `paragraph--callout-item.html.twig` file is just passing `callout__text: content.field_text` the text wrapper is still showing (because of Drupal's render array thing. Even though there's no content in the field, there are still items in the array - so it evaluates to true)

You'll need to update that to `callout__text: content.field_text[0]` to check whether there's actually content in the field. Then, this should work.

Reply:
@ModulesUnraveled I have added a branch to fix this here: https://github.com/yalesites-org/atomic/pull/60

Thank you!